### PR TITLE
node selector, file system update

### PR DESCRIPTION
**Pull request by rusenask**

### CODE REVIEW

File: _docs/manage/volumes/index.md [hunk 1]
```diff
@@ -129,3 +129,22 @@ default/volume-name
...
+
+```bash
+$ storageos volume create --nodeSelector 'country=US'
+```
+
+Another options for this case:
+
+```
+country in (US, CA)
+country notin (UK)
```
Comment (oicheryl) [inline _docs/manage/volumes/index.md]: If this syntax the same as K8s selectors, please link to https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors.

File: _docs/manage/volumes/index.md [hunk 2]
```diff
@@ -129,3 +129,22 @@ default/volume-name
 
 Volumes may not be removed immediately as the data will be purged in the
 background.
+
+## Using node selector
+
+Node selector is a form of constraint. For the volume to be eligible for a deployment on a node, selector must match node's labels. An empty selector
```
Comment (oicheryl) [inline _docs/manage/volumes/index.md]: "A node selector is a constraint on where volumes may be deployed, based on node labels."

File: _docs/manage/volumes/index.md [hunk 3]
```diff
@@ -129,3 +129,22 @@ default/volume-name
...
+always matches a node.
+
+If a group of nodes have a label `country=US`, then a node selector could be used
```
Comment (oicheryl) [inline _docs/manage/volumes/index.md]: "have a label" => "are labelled"

Comment (oicheryl) [reply _docs/manage/volumes/index.md]: "could" => "can"

File: _docs/manage/volumes/index.md [hunk 4]
```diff
@@ -129,3 +129,22 @@ default/volume-name
...
+during volume creation:
+
+```bash
+$ storageos volume create --nodeSelector 'country=US'
+```
+
+Another options for this case:
```
Comment (oicheryl) [inline _docs/manage/volumes/index.md]: Selectors may also be set-based:

**Comment by croomes** [comment]
Thanks, merged

**Pull request merged**